src/models/mongo/bulkTransfer.js

```diff
@@ -0,0 +1,214 @@
+/*****
+ License
+ --------------
+ Copyright © 2017 Bill & Melinda Gates Foundation
+ The Mojaloop files are made available by the Bill & Melinda Gates Foundation under the Apache License, Version 2.0 (the "License") and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ * Georgi Georgiev <georgi.georgiev@modusbox.com>
+ * Valentin Genev <valentin.genev@modusbox.com>
+ --------------
+ ******/
+'use strict'
+
+const mongoose = require('mongoose')
+
+// single transfer model
+const transfer = {
```

Comment:
I almost feel like this should go somewhere else under a `/schema` folder somewhere. Let me know what you think?

Reply:
Yes, I like the idea.

Reply:
How about `src/bulkApi/lib/mongodb.js` and `src/bulkApi/lib/queries.js`?
Also, what do you think about having:
`src/schema/definition/bulkTransfer.js` and
`src/schema/query/{collectionName}.js`?

Reply:
I think you misunderstood.

The bulkTransfer should be part of `src/model`. It is not a schema. The inline `transfer` object is a schema definition, or am I miss-reading this?

The comment above is referring to the `transfer` inline schema above...suggesting that maybe it should go in a folder like `schema` somewhere for example `src/models/schemas`. 

Not required, but may not be a bad idea if we are using the same schema elsewhere. If it is only being used in the `bulkTransfer`, then let's keep it there for the time being.

Reply:
Just combining all my original comments :

`src/models/mongo/bulkTransfer.js` --  just remove the `mongo` from the path --> `src/models/bulkTransfer.js`

This one, in my opinion, is completely optional: The inline `transfer` object schema definition in the file could be moved out to a  schema folder like --> `src/models/schemas/transfer.js`

Reply:
Thanks for helping out putting it into the library